### ISSUE TITLE
Incorrect Craft in Default Zone Notes

### ISSUE THREAD

**Issue by WolfgangSho** [JusKillmeQik/PoE-Leveling-Guide#98]
Hi , this is super minor but thought I'd let you know.

In The Crematorium (Act 3, third zone), the notes mention there being a Fire Damage crafting recipe. However, when I went to pick it up, it gave me Defences.

If you check here: http://www.vhpg.com/poe-crafting-recipes/ you can confirm the recipe in Crematorium is in fact Defences.

If you would like me to run through all the default notes and bring them up to speed, I'd be happy to make a pull request. It is defo boring legwork so it would be the least I could do for such an awesome overlay!

PS. Just checked and it says Fire Damage in the Detailed notes as well.

**Comment by JusKillmeQik** [comment]
Yeah, if you could change it and make a pull request that would be great

**Comment by JusKillmeQik** [comment]
Incorrect crafts have been fixed, we're still missing some of the newer crafts around sockets.
